### ISSUE TITLE
Additional output formats

### ISSUE THREAD

**Issue by michel-kraemer** [Juris-M/citeproc-js#70]
A user of citeproc-java has expressed the wish that the additional output formats I included in citeproc-java (AsciiDoc and FOP) should be moved to citeproc-js:
https://github.com/michel-kraemer/citeproc-java/issues/35

What do you think about this? Should I prepare a pull request?

**Comment by fbennett** [comment]
Absolutely no objection, that would be very welcome.

**Comment by dominic01** [comment]
Could you please let us know how to get this output in citeproc-js? A demo script would be great help.

**Comment by michel-kraemer** [comment]
@dominic01 You can select the output format as follows:

```js
var citeproc = new CSL.Engine(citeprocSys, styleAsText);
citeproc.setOutputFormat("html");
...
```

After my pull request has been merged, you will also be able to use `asciidoc` and `fo` as output formats.

**Comment by fbennett** [comment]
THis has been languishing for a long while, but I have merged the styles locally, and a fresh processor release should be out within a couple of days that can produce `asciidoc` or `fo` output.

**Comment by fbennett** [comment]
Fixed in tagged release 1.1.211.
